### ISSUE TITLE
IpDb strange prompt

### ISSUE THREAD

**Issue by Pitometsu** [realgud/realgud#210]
Hello. I have
```elisp
 '(realgud:ipdb-command-name
   "python -c \"from IPython import start_ipython; from sys import argv; start_ipython(['--simple-prompt', '-i', '-c', '%run -d {}'.format(' '.join(argv[1:]))])\"")
```
Note the `--simple-prompt` argument.
But when I call `M-x realgud:ipdb` I have mess in prompt:
```python
Current directory: /Users/netsu/Projects/csr/csr/ic/
Command: ipdb /Users/netsu/Projects/csr/csr/ic/nodes.py
> /Users/netsu/Projects/csr/csr/ic/nodes.py(1)<module>()
----> 1 import pydot
      2 import matplotlib
      3 matplotlib.use('TkAgg')

[6n[Jipdb> [6D[6C[6D[J[6n[Jipdb> [6D[6C[6D[J[6n[Jipdb> [6D[6C[6D[J[6n[Jipdb> [6D[6C[6D[J[6n[Jipdb> [6D[6C[6D[J[6n[Jipdb> [6D[6C[6D[J[6n[Jipdb> [6D[6C[6D[J[6n[Jipdb> [6D[6C[6D[J[6n[Jipdb> [6D[6C[6D[J[6n[Jipdb> [6D[6Cimport sys
[6D[Jipdb> import sys[16D
[J[6n[Jipdb> [6D[6Csys.argv
[6D[Jipdb> sys.argv[14D
[J['/Users/netsu/Projects/csr/csr/ic/nodes.py']
[6n[Jipdb> [6D[6Csys.executable
[6D[Jipdb> sys.executable[20D
[J'/Users/netsu/.local/share/virtualenvs/csr-44kfYsMA/bin/python2.7'
[6n[Jipdb> [6D[6C[6D[J[6n[Jipdb> [6D[6C[6D[J[6n[Jipdb> [6D[6C[6D[J[6n[Jipdb> [6D[6C[6D[J[6n[Jipdb> [6D[6C[6D[J[6n[Jipdb> [6D[6C[6D[J[6n[Jipdb> [6D[6C[6D[J[6n[Jipdb> [6D[6C[6D[J[6n[Jipdb> [6D[6C[6D[J[6n[Jipdb> [6D[6C[6D[J[6n[Jipdb> [6D[6C[6D[J[6n[Jipdb> [6D[6C[6D[J[6n[Jipdb> [6D[6C[6D[J[6n[Jipdb> [6D[6C[6D[J[6n[Jipdb> [6D[6C[6D[J[6n[Jipdb> [6D[6C
````

**Comment by rocky** [comment]
Have you looked at https://github.com/realgud/realgud/wiki/ipdb-notes ?

**Comment by Pitometsu** [comment]
Yes, and I use `--simple-prompt`. Don't want to create additional executables as a hacky solution.

**Comment by rocky** [comment]
See also https://github.com/realgud/realgud/wiki/ANSI-Terminal-Escape-appearing-in-Buffers

**Comment by rocky** [comment]
And also try: 

```
(autoload 'ansi-color-for-comint-mode-on "ansi-color" nil t)
(add-hook 'shell-mode-hook 'ansi-color-for-comint-mode-on)
```

**Comment by rocky** [comment]
Fixed with 36782d69c806079eb067705ef67fcb3b9299ae01
